content/install_maintain/installation/manager-image.md

```diff
@@ -44,16 +44,20 @@ Make sure that your environment meets the [prerequisites]({{< relref "install_ma
         This Docker image is designed to run on a remote instance, not on a docker installation on your local machine.
```

Comment:
The whole Docker section should be indented to the same level of the "Setup an Instance of a Cloudify Manager Image".
Originally when we asked the user to download the file and then load the container it maded sense to group them. now it doesnt make sense anymore.

Reply:
Are you sure?  _Docker_ section is on the same level as _OpenStack_.  Both are a second step of _Setup an Instance of a Cloudify Manager Image_ section.

Q: What would you like to do about _OpenStack_ section?  Leave it as is or change the indentation as well?

Q: What would you like to do about third step of _Setup an Instance of a Cloudify Manager Image_ section (`To use Cloudify Manager from the terminal using the Cloudify CLI, run the following command with your instance details.`)?

Reply:
Closing this one, and will handle the indentation as a separate PR.